src/main/java/org/candlepin/subscriptions/marketplace/MarketplaceProperties.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2020 Red Hat, Inc.
```

Comment:
2021!  Can't wait until we're using that spotless plugin...I've got it configured to do all the headers as part as the format task :)